### PR TITLE
Allow dependabot to keep github actions up-to-date

### PR DESCRIPTION
**Pull request by jlosito**
This change allows dependabot to check any GitHub action which this project uses and submit PRs with version bumps in order to keep packages up-to-date on a weekly basis.

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/keeping-your-actions-up-to-date-with-dependabot

### CODE REVIEW

**Comment by x4m** [comment]
Looks cool, thanks!
@g0djan @reshke @usernamedt what do you think?

**Review by g0djan — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (usernamedt) [inline .github/dependabot.yml:6]:
```suggestion
      interval: "weekly"
  - package-ecosystem: "gomod"
    directory: "/"
    schedule:
      interval: "weekly"
```

Comment (usernamedt) [reply .github/dependabot.yml:6]: I suggest adding the go modules dependencies check too

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (usernamedt) [inline .github/dependabot.yml:10]:
```suggestion
      interval: "weekly"
    allow:
      - dependency-name: "github.com/wal-g/tracelog"
      - dependency-name: "github.com/wal-g/storages"
```

Comment (usernamedt) [reply .github/dependabot.yml:10]: Limit go modules dependency alerts only for wal-g repositories.

**Review by usernamedt — APPROVED** [review]

**Pull request merged**